CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 2.22.0-rc.0 / 2020-10-07
+
+* [CHANGE] web: Remove APIv2. #7935
+* [FEATURE] React UI: Implement missing TSDB head stats section. #7876
+* [ENHANCEMENT] UI: Clarify alert state toggle via checkbox icon. #7936
+* [ENHANCEMENT] Add tsdb startup duration metric. #7737
+* [ENHANCEMENT] Include GOOS and GOARCH in version. #7968
```

Comment:
This is too minor to mention, the only people who will notice are this.

Reply:
Removed